src/modules/notifications/selectors/notification-state.js

```diff
@@ -100,6 +117,113 @@ export const selectMarketsInDispute = createSelector(
   }
 );
 
+// Get reportingFees for signed in user
+export const selectUsersReportingFees = createSelector(
+  selectReportingWindowStats,
+  reportingWindowStats => {
+    if (reportingWindowStats && reportingWindowStats.reportingFees) {
+      const { unclaimedEth, unclaimedRep } = reportingWindowStats.reportingFees;
+      if (
+        (unclaimedEth && unclaimedEth.value > 0) ||
+        (unclaimedRep && unclaimedRep.value > 0)
+      ) {
+        return reportingWindowStats.reportingFees;
+      }
+    }
+    return {};
+  }
+);
+
+// Get all unsigned orders from localStorage
+export const selectUnsignedOrders = createSelector(
+  selectMarkets,
+  selectLoginAccountAddress,
+  (markets, userAddress) => {
+    if (localStorageRef && userAddress) {
+      const getUserState = JSON.parse(localStorageRef.getItem(userAddress));
```

Comment:
there is a reducer that manages this data.

`loadPendingLiquidityOrders` in src/modules/auth/actions/load-account-data-from-local-storage.js

Also, the modal will clear liquidity orders depending if user cancels or sends. Also the direction is to consolidate interaction with localStorage.

Reply:
fixed! using the selector for this now.